endpoints/publicProxy/http.go

```diff
@@ -360,6 +364,23 @@ func SetZrokCookie(w http.ResponseWriter, cookieDomain, email, accessToken, prov
 	})
 }
 
+func deleteZrokCookies(w http.ResponseWriter, r *http.Request) {
+	// Get all cookies from the request
+	cookies := r.Cookies()
+	// Clear the Cookie header
+	r.Header.Del("Cookie")
+	// Save cookies not in the list of cookies to delete, the pkce cookie might be okay to pass along to the HTTP
+	// backend, but zrok-access is not because it can contain the accessToken from any other OAuth enabled shares, so we
+	// delete it here when the current share is not OAuth-enabled. OAuth-enabled shares check the audience claim in the
+	// JWT to ensure it matches the requested share and will send the client back to the OAuth provider if it does not
+	// match.
+	for _, cookie := range cookies {
+		if cookie.Name != "zrok-access" || cookie.Domain != "pkce" {
+			r.AddCookie(cookie)
```

Comment:
Shouldn't this have the `-1` thing to delete the cookie? It looks like you're just re-adding the same cookie?

Reply:
`cookie.MaxAge=-1` would be applicable if we were processing the response to the client, but this is a request on its way to the zrok backend, so the client won't have an opportunity to enforce that max age by deleting the cookie.

Reply:
`cookie.MaxAge=-1` would be applicable if we were processing the response to the client, but this is a request on its way to the zrok backend, so the client won't have an opportunity to enforce that max age by deleting the cookie.